apps/omg_watcher/lib/exit_processor.ex

```diff
@@ -465,21 +467,19 @@ defmodule OMG.Watcher.ExitProcessor do
     spend_blknum
   end
 
-  defp run_block_getting(%ExitProcessor.Request{blknums_to_get: blknums} = request) do
+  defp run_block_getting(%ExitProcessor.Request{blknums_to_get: blknums} = request),
+    do: %{request | blocks_result: do_block_getting(blknums)}
+
+  defp run_ife_block_getting(%ExitProcessor.Request{piggybacked_blknums_to_get: blknums} = request),
```

Comment:
Are the blknums piggybacked?
I understand the idea, nevertheless the name is quite not accurate.

Reply:
what about `fetching` instead of `getting`

Reply:
I don't like the name either, but let's defer this to a larger refactor of EP (which hopefully will come at one point :clock1:)